epam_tr/src/main/java/com/epam/tr/service/logic/validator/UserValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.tr.service.logic.validator;
+
+import com.epam.tr.entities.AppUser;
+import com.epam.tr.exceptions.InvalidFileException;
+import org.springframework.stereotype.Component;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+@Component
+public class UserValidator implements Validator<AppUser> {
+
+    @Override
+    public boolean isValid(AppUser entity) {
+        if(!(!isNullOrEmpty(entity.getLogin()) && !isNullOrEmpty(entity.getPassword()))) {
+            String message = String.format("App user :%s is not valid",entity.toString());
```

Comment:
for this case the message is ok, because we dont want to say user what exactily is wrong.